docker/README.md

```diff
@@ -32,6 +32,21 @@ for more information on commands.  Note also the 'run' makefile target which
 should simplify the invocation by guaranteeing that the parameters are consistent
 with what was used during `test-env-setup`, just invoke as `make run ARGS='<docker-compose subcommand & args>'`
 
+If you define the DEBUG environment variable, the make commands will (with help of `docker/sawtooth.debugging.yaml') build
```

Comment:
This should be PDO_DEBUG_BUILD, right?

Reply:
yep, left-over from when it still was DEBUG. Pushed corresponding fix.